documentation/airport-guides/lszh.md

```diff
@@ -0,0 +1,51 @@
+### About this facility
+Skyguide Air Navigation Center, Dübendorf, Switzerland
+269,160 annual operations in 2016
+
+Located at Dübendorf Air Base (close to Zurich), besides the LSAZ FIR, the Skyguide Air Navigation Center is also responsible for the in- and outbound traffic of Switzerlands major airport Zurich (LSZH).
+
+### Airspace
+The airspace area extends up to 55nm of distance to the airport and is also covering parts of German national territory. Due to the fact that approaches to runway 14 and 16 require the use of German airspace, these approaches aren't available during the German ordinance period. (MON - FRI: 2100-0700LT; SAT, SUN, holidays;  2000-0700LT). Therefore runway 28 or 34 is usually used for landings then which is reducing the capacity of the airport. Vertically the airspace is covering aircraft up to FL190.
+
+### Instrument Procedures
+For outbound traffic LSZH offers a couple of SIDs eventhough usually only DEGES and VEBIT are used. Inbound traffic follows a STAR to either GIPOL or AMIKI. Inbound traffic must be vectored to their final approach course.
+
+### Sectors Available in OpenScope
+```
+D - Zurich Departure
+W - Zurich Arrival (West)
+E - Zurich Arrival (East)
+A - Zürich Arrival (ARFA)
+F - Zurich Final Approach
+```
+
+### Airports of Interest
+
+#### LSZH (Primary Airport)
+Zurich airport 
+269,160 annual operations
+
+Zürich Airport, also known as Kloten Airport, is the largest international airport of Switzerland and the principal hub of Swiss International Air Lines. It serves Zürich, Switzerland's largest city, and, with its surface transport links, much of the rest of the country. The airport is located 13 kilometres (8 mi) north of central Zürich, in the municipalities of Kloten, Rümlang, Oberglatt, Winkel and Opfikon, all of which are within the canton of Zürich.
+
+
+#### EDNY
+Flughafen Friedrichshafen (Friedrichshafen Airport)
+
```

Comment:
Please add annual operations (approximate) and distance from LSZH to the other airports